### DISCUSSION TITLE
Add parameter on Iden to automatically add table name to column name

### DISCUSSION THREAD

**Discussion by Sytten** [SeaQL/sea-query#104] [Ideas]
Hello!
Most of the time you want to express a column name relative to the parent. You can co that with `Expr::tbl` but it does create a lot of unnecessary boilerplate since we already know the table in the enum.
My suggestion would be to add an option to Iden that adds the name of the table automatically so you can use `Expr::col` and not worry about it. Since this would be totally optional, we could decide to not use it if you don't want to.
I can implement it if this is approved by the maintainer :)
Thanks

**Comment by tyt2y3** [comment]
That actually is the default behavior of [SeaORM](https://github.com/SeaQL/sea-orm), i.e. to prepend the table name to the column name.
SeaQuery is the lower level library, and I don't think it is the best place to offer this feature.
